runner/src/main/scala/se/kth/benchmarks/runner/Runner.scala

```diff
@@ -16,7 +16,7 @@ object Runner {
   type Stub = BenchmarkRunnerGrpc.BenchmarkRunnerStub;
 
   val WAIT: Duration = 500.milliseconds;
-  val MAX_RETRIES: Int = 30;
+  val MAX_RETRIES: Int = 130;
```

Comment:
What's with that? Do you really want to wait 2min before the runner realises that some node can't be connected to?

Reply:
30*500ms was not enough for some benchmarks to init properly. My most recent run had a benh retry 45 times before succeeding, on my workstation, so 30 is definitely too low.

Reply:
Really? Gee, that's very long.

Reply:
Same for me, sometimes mine starts to run on 120th attempt...